atom/common/options_switches.cc

```diff
@@ -113,6 +113,10 @@ const char kDisableHttpCache[] = "disable-http-cache";
 // Register schemes to standard.
 const char kRegisterStandardSchemes[] = "register-standard-schemes";
 
+// The minimum SSL/TLS version ("tls1", "tls1.1", or "tls1.2") that
+// TLS fallback will accept.
+const char kSSLVersionFallbackMin[] = "ssl-version-fallback-min";
```

Comment:
i believe providing `ssl-version-max` is not the right one, as chromium already fallbacks with lesser TLS versions for buggy servers but since chromium 44 the fallback version was restricted to "tls1.1" , so this should enable to set a lower fallback https://www.chromium.org/administrators/policy-list-3#SSLVersionFallbackMin